extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php

```diff
@@ -14,9 +14,6 @@
 
 class WikiaHomePageHelper extends WikiaModel {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "WikiaHomePageHelper" has 41 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)